man/tmpfiles.d.xml

```diff
@@ -585,9 +585,31 @@ w- /proc/sys/vm/swappiness - - - - 10</programlisting></para>
       <para>The age of a file system entry is determined from its last
       modification timestamp (mtime), its last access timestamp (atime),
       and (except for directories) its last status change timestamp
-      (ctime). Any of these three (or two) values will prevent cleanup
-      if it is more recent than the current time minus the age
-      field.</para>
+      (ctime). By default, any of these three (or two) values will
+      prevent cleanup if it is more recent than the current time minus
+      the age field. To restrict the deletion based on particular type
+      of file timestamps, the age-by argument can be used.</para>
+
+      <para>The age-by argument, when (optionally) specified along
+      with age will check if the file system entry has aged by the
+      type of file timestamp(s) provided. It can be specified by
+      prefixing the age argument with a set of file timestamp types
+      followed by a colon character <literal>:</literal>, i.e.,
+      <literal><replaceable>age-by</replaceable>:<replaceable>cleanup-age</replaceable></literal>.
+      The argument can be a set of:
+      <constant>a</constant> (<constant>A</constant> for directories),
+      <constant>b</constant> (<constant>B</constant> for directories),
+      <constant>c</constant> (ignored for directories), or
+      <constant>m</constant> (<constant>M</constant> for directories),
+      indicating access, creation, last status change, and last
+      modification times of a file system entry respectively. See
+      <citerefentry project='man-pages'><refentrytitle>statx</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+      file timestamp fields for more details.</para>
```

Comment:
Maybe add:

"If not specified the age-by field defaults to abcmABM, i.e. by default all timestamps are taken into consideration, with the one exception of the change time (ctime) for directories. This is because the aging logic itself will alter the ctime whenever it deletes a file inside it. To ensure that running the aging logic does not feed back into the next iteration of it, ctimes for directories are ignored by default." or something like that

Reply:
Sounds good. ACK.